src/components/Layout/RightSidebar.tsx

```diff
@@ -30,8 +30,10 @@ const externalLinks = (
 
   if (currentPage) {
     const githubPathName = location.pathname.replace('docs/', '');
+    const githubPathSegments = githubPathName.split('/').filter((segment) => segment !== '');
     githubEditPath =
-      githubBasePath + (currentPage.indexPage ? `${githubPathName}/index.textile` : `${githubPathName}.textile`);
+      githubBasePath +
+      (githubPathSegments.length === 1 ? `${githubPathName}/index.textile` : `${githubPathName}.textile`);
```

Comment:
This works as there is a common convention across the template file organisation in github that only top level overview pages have `/index.textile` files - everything else at every further depth has a named template.

Reply:
For certain. After the MDX work, I don't want to have to do any DOM scraping or path parsing, but the current setup is too inflexible for that now